OpenRA.Mods.Common/Widgets/ButtonWidget.cs

```diff
@@ -224,6 +224,10 @@ public override void MouseExited()
 			tooltipContainer.Value.RemoveTooltip();
 		}
 
+		public string Cursor = ChromeMetrics.Get<string>("ButtonCursor");
```

Comment:
Please define this at the top of the class with all the other Button configuration options.

Reply:
Okay.